### ISSUE TITLE
BUG: Typo in Examples Page?

### ISSUE THREAD

**Issue by jsphon** [fchollet/keras#1670]
In the examples on http://keras.io/examples/, I think the lines that read

    from keras.layers import Dense, Dropout, Activation

should be

    from keras.layers.core import Dense, Dropout, Activation

If not, there is a problem with my installation.;P

**Comment by farizrahman4u** [comment]
Try the latest version of Keras. You can import all core layers from `keras.layers`.

**Comment by jsphon** [comment]
Thanks, that worked.

For reference, I cloned the repo with this commad:

`pip install --upgrade --no-deps git+git://github.com/fchollet/keras.git`
